src/PointwiseFunctions/AnalyticSolutions/GrMhd/SmoothFlow.hpp

```diff
@@ -151,14 +146,18 @@ class SmoothFlow {
       std::numeric_limits<double>::signaling_NaN();
   PerturbationAmplitude::type perturbation_amplitude_ =
       std::numeric_limits<double>::signaling_NaN();
+  // The angular frequency.
+  double k_dot_v_ = std::inner_product(
```

Comment:
I don't see how this can work here...

Reply:
Initialization occurs in order as far as I know, and this would be called at the end of the list initialization. @wthrowe is my memory correct?

Reply:
won't you get an FPE if it uses the default values above?

Reply:
I believe all of these statements are true, and in fact I observe an FPE if I run the test on my machine (when default-constructing in the option parser, it looks like).

Reply:
Ah, yep. The default constructor would blow up... :(